src/components/twilio-voice-dialer/routes.js

```diff
@@ -0,0 +1,64 @@
+import { Router } from 'express';
+import crypto from 'crypto';
+import Twilio from 'twilio';
+import config from '../../config.js';
+
+const AccessToken = Twilio.jwt.AccessToken;
+const VoiceGrant = AccessToken.VoiceGrant;
+const VoiceResponse = Twilio.twiml.VoiceResponse;
+
+const router = Router();
+const { appSid, accountSid, apiKeySid, apiKeySecret, callerId, callbackBaseURL } = config;
+const client = Twilio(apiKeySid, apiKeySecret, { accountSid });
+
+router.get('/token', (req, res) => {
+  const identity = req.query.identity;
+  const token = new AccessToken(accountSid, apiKeySid, apiKeySecret, { identity, ttl: 3600 });
+  const voiceGrant = new VoiceGrant({
+    outgoingApplicationSid: appSid,
+    incomingAllow: !!identity,
+  });
+
+  token.addGrant(voiceGrant);
+  res.send({ token: token.toJwt() });
+});
+
+router.post('/twiml', (req, res) => {
+  const twiml = new VoiceResponse();
+  const dial = twiml.dial();
+  
+  // Generates 1:1 conference
```

Comment:
Why does the `/twiml` route handle the conference logic?

Reply:
They are not mutually exclusive. TwiML is an instruction. We are 'instructing' Twilio to create a conference to dial this recipient.